src/lib/generator.ts

```diff
@@ -19,16 +19,32 @@ export interface OutputFile {
   content: string;
 }
 
-/*
- * TODO (justinfagnani): we need a type / parameter to represent defined
- *     elements, so that when they're encountered we can generate the correct
- *     Soy call command to their template.
+/**
+ * A map of custom element tag names to the name of their class.
  */
+export type DefinedElements = {
```

Comment:
I don't think we're actually using definedElements anywhere yet, so let's remove it as YAGNI.

Reply:
Cleared out definedElements in 1d9fb19. definedElements never contains anything except for in one test so I removed it all. If you just meant remove it from the generator interface and leave it in the converter, let me know